CHANGELOG.md

```diff
@@ -1,128 +1,130 @@
+- We've made `USpatialMetrics::WorkerMetricsRecieved` static.
+- You can now connect to a local deployment by selecting "Connect to a local deployment" and specifying the local IP address of your computer in the Launch drop-down menu.
+- We've enabled RPC ring buffers by default, and we'll remove the legacy RPC mode in a future release.
+- We've removed the `bPackRPCs` property and the the `--OverrideRPCPacking` flag.
+- We've added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and Actor Components. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
+- You can now generate valid schema for classes that start with a leading digit. The generated schema class are prefixed with `ZZ` internally.
+- The GDK now automatically replicates handover properties when this is required for load balancing. `bEnableHandover` is off by default.
+- We've added `OnSpatialPlayerSpawnFailed` delegate to `SpatialGameInstance`. This is helpful if you established a successful connection but the server-worker instance crashed.
 - The GDK now uses SpatialOS 14.6.1.
-- Add ability to disable outgoing RPC queue timeouts by setting `QueuedOutgoingRPCWaitTime` to 0.0f.
-- Added `bWorkerFlushAfterOutgoingNetworkOp` (defaulted false) which publishes changes to the GDK worker queue after RPCs and property replication to allow for lower latencies. Can be used in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency at a trade-off with bandwidth.
-- You can now edit the project name field in the `Cloud Deployment` window.
-- Worker types are now defined in the runtime settings.
-- Local deployment will now use the map's load balancing strategy to get the launch configuration settings. The launch configuration file is saved per-map in the Intermediate/Improbable folder.
-- A launch configuration editor has been added under the Deploy toolbar button.
-- The cloud deployment window can now generate a launch configuration from the current map or use the launch configuration editor.
+- You can now disable outgoing RPC queue timeouts by setting `QueuedOutgoingRPCWaitTime` to 0.0f.
+- We've added `bWorkerFlushAfterOutgoingNetworkOp` (default false) which publishes changes to the GDK worker queue after RPCs and property replication, to allow for lower latencies. You can use this in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency at a trade-off with bandwidth.
+- You can now edit the project name field in the Cloud Deployment window.
+- You now define worker types in the Runtime Settings.
+- Local deployments now use the map's load balancing strategy to get the launch configuration settings. The launch configuration file is saved per-map in the `Intermediate/Improbable` folder.
```

Comment:
Is "the map's LB strategy" the right way of putting it? As opposed to "the worker instance's LB strategy" or something else?

Reply:
Yes, Load balancing strategies are per map.